ReactWindows/ReactNative.Tests/Modules/Network/NetworkingModuleTests.cs

```diff
@@ -363,6 +365,7 @@ public void NetworkingModule_Response_Content_UseIncremental()
                     }
                     else if (eventName == "didCompleteNetworkResponse")
                     {
+                        onCompleteData = args[1] as JArray;
```

Comment:
would it be better to explicitly cast here so that the test would fail with InvalidCastException that includes the two types, rather than an NRE down below?